app/controllers/journey_controller.rb

```diff
@@ -30,8 +31,7 @@ def summary
     @location = Location.find(params[:location_id])
     @event_generator = RouletteService.new(params, current_character)
     status = @event_generator.generate_travel_event
-    
-    render layout: 'wide',  :locals => {:background => 'start'}
+    render layout: 'wide',  :locals => {:background => 'start', status: status}
```

Comment:
Unnecessary spacing detected.<br>Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.